src/rest_api/amoc_api_logic_handler.erl

```diff
@@ -13,6 +13,15 @@
 -spec handle_request(OperationID :: amoc_rest_api:operation_id(),
                      Req :: cowboy_req:req(), Context :: #{}) ->
     {Status :: cowboy:http_status(), Headers :: cowboy:http_headers(), Body :: jsx:json_term()}.
+handle_request('StatusGet', _Req, _Context) ->
+    Status = amoc_api_helpers_status:get_status(),
+    {200, #{}, Status};
+handle_request('StatusNodeGet', _Req, #{node := BinNode}) ->
+    Node = binary_to_atom(BinNode, utf8),
+    case rpc:call(Node, amoc_api_helpers_status, get_status, []) of
+        {badrpc, _} -> {404, #{}, #{}};
```

Comment:
Can you please add some description to the error?

Reply:
it is atready described in openapi spec